include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
Can we do the vector<bool> case by SFINAEing on whether we can do &v[0] or not? Then we don't need any `#include <vector>`.